docs/Contributing/architecture/software/README.md

```diff
@@ -0,0 +1,21 @@
+# Software Architecture
+
+This directory contains documentation about Fleet's Software architecture.
+
+## Overview
+
+Fleet's Software architecture is designed to manage software across the device fleet, including software inventory, vulnerability management, and software installation. This documentation provides insights into the design decisions, system components, and interactions specific to the Software functionality.
+
+## Contents
+
+- [Software Overview](software-overview.md) - Overview of Fleet's Software architecture
+- [Software Inventory](software-inventory.md) - Architecture for software inventory
+- [Vulnerability Management](vulnerability-management.md) - Architecture for vulnerability management
+- [Software Installation](software-installation.md) - Architecture for software installation
+- [Software Updates](software-updates.md) - Architecture for software updates
+- [Software Policies](software-policies.md) - Architecture for software policies
```

Comment:
Per current responsibilities, policies are primarily orchestration's responsibility, so policies docs should be there rather than here. We can cover policy automations somewhere in these docs, but they probably don't need their own file.

Reply:
Updating.